lib/Debugger.ProtocolHandler/DebuggerImpl.cpp

```diff
@@ -470,6 +470,65 @@ namespace JsDebug
         }
     }
 
+    SkipPauseRequest DebuggerImpl::EvaluateConditionOnBreakpoint(int bpId)
+    {
+        if (bpId < 0)
+        {
+            return SkipPauseRequest::RequestNoSkip;
+        }
+
+        DebuggerBreakpoint *bp = nullptr;
+        auto it = m_breakpointMap.begin();
+        while (it != m_breakpointMap.end())
+        {
+            if (it->second.GetActualId() == bpId)
+            {
+                bp = &it->second;
+                break;
+            }
+            it++;
+        }
+
+        if (bp == nullptr)
+        {
+            return SkipPauseRequest::RequestNoSkip;
+        }
+
+        try
+        {
+            String condition = bp->GetCondition();
+            if (condition.length() != 0)
+            {
+                DebuggerCallFrame callFrame = m_debugger->GetCallFrame(0);
+                std::unique_ptr<ExceptionDetails> exceptionDetails;
+                JsValueRef expressionStr = JS_INVALID_REFERENCE;
+                if (JsCreateStringUtf16(condition.characters16(), condition.length(), &expressionStr) == JsNoError)
+                {
+                    JsValueRef evalResult = JS_INVALID_REFERENCE;
+                    JsErrorCode err = JsDiagEvaluate(
```

Comment:
I would prefer not having each argument on a separate line. Not sure why it is done here but not other places. Can you update to just be on one line?

Reply:
will do.